client/src/components/Reviews/ReviewsAPIUtils.js

```diff
@@ -34,6 +34,11 @@ export default {
   },
 
   submitFormData: (formData, config) => {
-    return axios.post('api/reviews', formData, config);
+    return axios.post('/api/reviews', formData, config);
+  },
+
+  sendClickData: (clickData) => {
+    console.log('clickData:', clickData);
+    return axios.post('/api/interactions/clickData', clickData);
```

Comment:
great idea to make a new route for interactions!

Reply:
Agreed